src/main/java/mate/hwdao/controllers/AddDriverController.java

```diff
@@ -0,0 +1,37 @@
+package mate.hwdao.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.hwdao.Main;
+import mate.hwdao.lib.Injector;
+import mate.hwdao.model.Driver;
+import mate.hwdao.service.DriverService;
+
+public class AddDriverController extends HttpServlet {
+    private static final Injector injector =
+            Injector.getInstance(Main.class.getPackageName());
```

Comment:
Same here
![Screenshot 2021-01-20 at 19 49 11](https://user-images.githubusercontent.com/67925366/105214361-94679c80-5b58-11eb-87e1-219259a01dd5.png)

Reply:
I'm not so sure about that.